test/charm.py

```diff
@@ -0,0 +1,3 @@
+class Charm:
```

Comment:
That's a bit curious. We shouldn't need it.

Reply:
I had it documented in charm-structure.rst:

```
    * a module-level ``Charm`` class definition from ``lib/charm.py`` provided
      by a charm author is used.
```

This is to enforce that charm authors put their Charm class code into `lib/charm.py` so that `entrypoint.py` can always import the `charm` module and a class called `Charm` from it without having to handle custom module names such as `mysql_charm.py`.

So `test/charm.py` is imported by `entrypoint.py` because there is no author-specific charm module available during test execution. Here is what happens without it:

```
env | grep PYTHON
PYTHONPATH=/home/ubuntu/src/juju/charm-tech-sketch/test

python3 -m test_entrypoint
# ...
  File "/home/ubuntu/src/juju/charm-tech-sketch/test/test_entrypoint.py", line 11, in <module>
    import juju.entrypoint as ep
  File "/home/ubuntu/src/juju/charm-tech-sketch/juju/entrypoint.py", line 11, in <module>
    from charm import Charm # noqa
ModuleNotFoundError: No module named 'charm'
```